samples/crypto/rsa/prj.conf

```diff
@@ -1,5 +1,5 @@
-CONFIG_MAIN_STACK_SIZE=4096
-CONFIG_HEAP_MEM_POOL_SIZE=4096
+CONFIG_MAIN_STACK_SIZE=16384
+CONFIG_HEAP_MEM_POOL_SIZE=16384
```

Comment:
> The worst-case we have for this is psa_tls which (if using RSA and DHM) requires 56000 bytes of heap memory.

yes, and what i'm asking is that we somewhere document this.
- directly in the docs
- In Kconfig help text, though unsure on which settings
- As a comment in this `rsa/prj.conf`, like
   `# When RSA is enable, a minimal heap size of 16k is required by mbedTLS.`
- As a comment in the commit message, so that anyone doing a `git blame` to see why this was set to 16k, can see the commit message for the commit introducing this change, and thereby gain more knowledge.
   Using `git blame` and reading the reason for a given line change in the commit message is very helpful.

Feel free to do one or all of above, but make this clearer to anyone reading code / `git blame` as to why this value is 16k.

Reply:
Adding this in known issues.  Don't want to run CI again. Then this doesn't get into the release.